src/store.h

```diff
@@ -90,9 +98,9 @@ class MetadataStore {
                       const MonitoredResource& resource);
 
   // Updates metadata for a given resource.
-  // Adds a metadata mapping from the `resource` to the metadata `entry`.
-  void UpdateMetadata(const MonitoredResource& resource,
-                      Metadata&& entry);
+  // Adds a metadata mapping from the Full Resource Name to the metadata
```

Comment:
How about just ``Adds a metadata mapping from the `full_resource_name` to the metadata `entry`.``?

Reply:
Done